src/pongstar.cpp

```diff
@@ -43,6 +43,11 @@ void Pongstar::initialize(HWND hwnd) {
 }
 
 void Pongstar::initializeEntities() {
+	// RNG
+	std::random_device rd;     // only used once to initialise (seed) engine
+	std::mt19937 rng(rd());    // random-number engine used (Mersenne-Twister in this case)
+	std::uniform_int_distribution<int> uniformDirection(0, 1);
```

Comment:
rename to `randBool`? its called `uniform_int_distribution` because the distribution for the randomness of numbers is even